opl/opl_sdl.c

```diff
@@ -161,8 +178,11 @@ static void OPL_SDL_Shutdown(void)
     */
 }
 
-static int OPL_SDL_Init(unsigned int port_base)
+static int OPL_SDL_Init(unsigned int port_base, int *num_chips)
 {
+    num_opl_chips = *num_chips < 1 ? 1 : *num_chips > OPL_MAX_CHIPS ? OPL_MAX_CHIPS : *num_chips;
+    *num_chips = num_opl_chips;
```

Comment:
Same thing here, but maybe changes to the handling of `num_opl_chips` will simplify this anyway.